src/live.asp.net/wwwroot/js/site.js

```diff
@@ -6,6 +6,13 @@
 
     var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
 
+	var countdownPollInterval = 500;
+
+	var liveShowPollInterval = 15 * 1000;
+
+	var SHOW_STATUS = { "OffAir": 1, "OnAir": 2, "Standby": 3 };
+	Object.freeze(SHOW_STATUS);
```

Comment:
Have a look at From http://caniuse.com/#feat=es5 (click show all)
IMHO there is a good support for the full ECMAScript 5 specification on mobile.

Maybe you could have a look at the useragent stats of the website to help you decide, but as I said we could just drop the freeze and it will just work in the same way for this use case.